### ISSUE TITLE
Field's `onChange` gets triggered if any field changes

### ISSUE THREAD

**Issue by fromanother** [davidkpiano/react-redux-form#249]
https://jsfiddle.net/zp3r8okw/

In the above code we have two fields, one of them has an `onChange` handler, but it gets executed if any of the fields in the form are changed.

Is this desired behavior?

**Comment by davidkpiano** [comment]
Good catch! This is not the intended behavior, but the internal way that `onChange`, `onFocus`, etc. are being handled internally will be changed (and greatly improved) in the upcoming v1.0.

However, this might be a bug, so I'll see about patching it.

**Comment by elodszopos** [comment]
@davidkpiano Confirmed bug. I ran into this just two days ago.

**Comment by davidkpiano** [comment]
@elodszopos @fromanother Can you look at the above unit test and let me know if it matches your use cases? https://github.com/davidkpiano/react-redux-form/commit/70bd8315e8a1acdfac2252cd2f4ba78d8de8aa2f#diff-b43d85b01204324e63c810644e344418R1319

The unit test above is actually passing, though the current code isn't published yet. This problem might already be fixed 😌

**Comment by elodszopos** [comment]
PR incoming in a sec with one more test case. Confirmed that the bug is taken care of. @davidkpiano
